src/js/stores/AppsStore.js

```diff
@@ -295,7 +314,7 @@ QueueStore.on(QueueEvents.CHANGE, function () {
 AppDispatcher.register(function (action) {
   switch (action.actionType) {
     case AppsEvents.REQUEST_APPS:
-      storeData.apps = processAppsAndEmptyGroups(action.data.body);
+      storeData.apps = processAppsAndGroups(action.data.body);
```

Comment:
Please adjust the method as proposed in https://github.com/mesosphere/marathon-ui/pull/710/files/051500d7d487b8f39b1bc900b4910fd177db4ea3#r57156202 and rework the properties and getter respectively.